src/server.rs

```diff
@@ -684,4 +690,45 @@ impl WalletService for Arc<crate::wallet::Wallet> {
             "not implemented",
         ))
     }
+
+    async fn ripemd160(
+        &self,
+        request: tonic::Request<Ripemd160Request>,
+    ) -> std::result::Result<tonic::Response<Ripemd160Response>, tonic::Status> {
+        let message = request.into_inner().message;
+        let message: Vec<u8> = message
+            .ok_or_else(|| missing_field::<GetBmmHStarCommitmentRequest>("message"))?
```

Comment:
This is a mistake, you should use `missing_field::<Ripemd160Request>("message")`